tests/build/build.go

```diff
@@ -119,24 +110,36 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				Expect(err.Error()).To(ContainSubstring("Reference does not exist"))
 			}
 
-			// Delete created webhook from GitHub
-			hooks, err := f.AsKubeAdmin.CommonController.Github.ListRepoWebhooks(helloWorldComponentGitSourceRepoName)
-			Expect(err).NotTo(HaveOccurred())
-
-			for _, h := range hooks {
-				hookUrl := h.Config["url"].(string)
-				if strings.Contains(hookUrl, pacControllerHost) {
-					Expect(f.AsKubeAdmin.CommonController.Github.DeleteWebhook(helloWorldComponentGitSourceRepoName, h.GetID())).To(Succeed())
-					break
-				}
-			}
-
 			//Delete the quay image repo since we are setting delete-repo=false
 			_, err = build.DeleteImageRepo(imageRepoName)
 			Expect(err).NotTo(HaveOccurred(), "Failed to delete image repo with error: %+v", err)
 
 		})
 
+		validateChecks := func() {
+			var checkSuite *github.CheckSuite
+			timeout = time.Minute * 15
```

Comment:
I think 15 minutes is a long time to wait for `validateChecks` since we are already checking `PipelineRun should eventually finish` in a previous step.. don't think it should take more than 1 minutes .. WDYT?

Reply:
yeah I'm happy to shorten the time

Reply:
As it will validate if the PaC Check suite status to be 'completed' , sometimes it does need a longer time.

Reply:
@xinredhat do you mean after the pipelinerun finishes .. the check suite status will take upto 15 minutes in some cases?